tests/Admin/AdminTest.php

```diff
@@ -756,7 +756,7 @@ public function testGetPerPageOptions(): void
 
         $perPageOptions = $admin->getPerPageOptions();
 
-        $this->assertSame([10, 25, 50, 100, 250], $perPageOptions);
+        $this->assertSame([16, 32, 64, 128, 256], $perPageOptions);
```

Comment:
We should update the two constants of the AbstractAdmin instead of the tests.

This was the values of our persistent bundle
https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/c18eb59881d7ebd7aeca92de9bf9e74c0e812090/src/Model/ModelManager.php#L613
https://github.com/sonata-project/SonataDoctrineMongoDBAdminBundle/blob/c18eb59881d7ebd7aeca92de9bf9e74c0e812090/src/Model/ModelManager.php#L629

Reply:
The constants are private.

The values of those method were never used because are overridden in

https://github.com/sonata-project/SonataAdminBundle/blob/a8731dc05c4546d83149576ee446dc1cc1daf7db/src/Admin/AbstractAdmin.php#L700-L709

Reply:
> The values of those method were never used because are overridden in
> 
> https://github.com/sonata-project/SonataAdminBundle/blob/a8731dc05c4546d83149576ee446dc1cc1daf7db/src/Admin/AbstractAdmin.php#L700-L709

I remember that in some issue "business" values were preferred over "binary" values.
They were meant to be used, that's why it was fixed in the master branch in the same time that the deprecation of `getDefaultSortValues`.
Currently the values in master are `[10, 25, 50, 100, 250]`, the introduction of a constant to not use "magic numbers" shouldn't change the values.

Reply:
Well, to be fair, in master tests were changed here, but I don't find any comment about that:

https://github.com/sonata-project/SonataAdminBundle/commit/0353882955fafd25b200b9200615edd5670a5577#diff-988f567ac1553662391a6e5079ce038f8f58cf2f1e7aafba378ce38eb2b9aaecR718-R728

Reply:
Comments were added in the AbstractAdmin https://github.com/sonata-project/SonataAdminBundle/pull/7102/files#diff-0850515af24b81ce716f53b8eb0b71fae91dcbdcc88b28d95a83886504e1c8f4R2660-R2663 and maybe the comment were missing in the test, but there was updated accordingly to the value change in the AbstractAdmin.

This is why I posted https://github.com/sonata-project/SonataAdminBundle/pull/7102#issuecomment-824088482
To add a `NEXT_MAJOR` comment here: https://github.com/sonata-project/SonataAdminBundle/pull/7102#issuecomment-824088482
```
    private const DEFAULT_LIST_PER_PAGE_RESULTS = 32; // NEXT_MAJOR: Change to 25
    private const DEFAULT_LIST_PER_PAGE_OPTIONS = [16, 32, 64, 128, 256]; // NEXT_MAJOR: Change to [10, 25, 50, 100, 250]
```
I thought you said you'll modify them https://github.com/sonata-project/SonataAdminBundle/pull/7102/files#issuecomment-824092510